extensions/manifest.replacer/src/test/resources/features/with-manifests.json

```diff
@@ -0,0 +1,24 @@
+{
+    "id": "org.eclipse.osgi.technology.featurelauncher.extensions.manifest.replacer.test:with-manifests:1.0",
+    "name": "With manifests",
+    "description": "With Manifests Feature with empty extension config",
+    "complete": true,
+    "bundles": [
+        {
+            "id": "org.eclipse.osgi.technology.featurelauncher.extensions.manifest.replacer.test:test-bundle:0.0.1",
+            "eclipse.osgi.technology.manifest.replacer.Bundle-ManifestVersion": "2",
```

Comment:
manifest.replacer" : [
"Line1",
"Line2"
]

Or would thi better in a handler that add/Override files in general on a give path.?
Aso with an base64 option?